kiagnose/internal/checkup/checkup_test.go

```diff
@@ -281,6 +268,21 @@ func TestCheckupTeardownShould(t *testing.T) {
 	})
 }
 
+func testTeardownShouldFailWhenNamespaceDeletionFails(t *testing.T) {
+	testClient := newNormalizedFakeClientset()
+	testCheckup := checkup.New(testClient, testCheckupName, &config.Config{Image: testImage, Timeout: testTimeout}, nameGeneratorStub{})
+
+	testClient.injectResourceVersionUpdateOnNamespaceCreation()
+
+	assert.NoError(t, testCheckup.Setup())
+
+	const expectedErr = "failed to delete ClusterRoleBinding"
```

Comment:
@ormergi I think we have a bug in this line.
I think it should be:
```suggestion
	const expectedErr = "failed to delete Namespace"
```

Reply:
At line 280 we configure the tests client with the error it will return on namespace delete,
Line 282 calls Teardown() which sends a delete namespace using that client.
Note that there is no actual bug, the error content doesn't have meaning in this test it could be set to "foo".
But yes the current message is wrong.

Reply:
Would you mind to send a PR to fix it?